app/package-lock.json

```diff
@@ -14,13 +14,16 @@
         "axios-oauth-client": "^1.4.4",
         "axios-token-interceptor": "^0.2.0",
         "body-parser": "^1.19.0",
+        "carbone": "^3.2.3",
```

Comment:
Why are we adding carbone to this app? The showcase app shouldn't need this at all.